test/e2e/util.go

```diff
@@ -333,7 +341,7 @@ func kubectlCmd(args ...string) *exec.Cmd {
 	kubectlArgs := append(defaultArgs, args...)
 
 	//TODO: the "kubectl" path string might be worth externalizing into an (optional) ginko arg.
-	cmd := exec.Command("kubectl", kubectlArgs...)
+	cmd := exec.Command(filepath.Join(testContext.RepoRoot, "cluster/kubectl.sh"), kubectlArgs...)
```

Comment:
OK, I will revert this change and open an issue (or maybe a bug) to make sure that gke/gke-test use the gcloud version instead. It's probably best to fix that in a separate PR.